project/plugins.sbt

```diff
@@ -2,13 +2,13 @@ resolvers += Resolver.url("hmrc-sbt-plugin-releases", url("https://dl.bintray.co
 resolvers += "Typesafe Releases" at "https://repo.typesafe.com/typesafe/releases/"
 resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.2.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.11.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.10.0")
 
-addSbtPlugin("uk.gov.hmrc"       % "sbt-distributables" % "1.6.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin"         % "2.6.25")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.7.5")
```

Comment:
I think this might be for bp-27, so it might be safer to use the 2.6.25 that was used before.

Reply:
what is bp-27?

Reply:
Sorry about that. I meant bootstrap 27

Reply:
the tests run locally and so does the service, if they are failing on your machine then we should upgrade it as much as possible otherwise theirs no reason not to.